code/final_code.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/python3
 import pygame
 import time
-from RPi import GPIO
+import gpiozero as gpio
```

Comment:
Should I leave it as `gpiozero`, shorten it to `gpio` or just import the `Button` class? What would work for clarity?